kubernetes/Service.yml

```diff
@@ -91,6 +91,24 @@ spec:
 ---
 apiVersion: v1
 kind: Service
+metadata:
+  name: freeradius
+  labels:
+    app: freeradius
+spec:
+  type: ClusterIP
```

Comment:
Need to think about how the Freeradius service will be reachable outside the cluster.  First thought is make the service Type = 'NodePort' but using 'externalIPs' may be enough.  Also, any multi-node cluster will need some external load balancing and most of the public clouds do not support UDP in their elastic load balancers.

Reply:
Thanks for informing, I did not know about Loadbalancers not supporting [UDP for most providers](https://kubernetes.io/docs/concepts/services-networking/service/#udp).

Reply:
Closing this as per the conversation on Gitter regarding all major providers supporting UDP now! :smile: